src/maths/shapes/Ellipse.ts

```diff
@@ -111,7 +111,7 @@ export class Ellipse implements ShapePrimitive
     // #if _DEBUG
     public toString(): string
     {
-        return `[@pixi/math:Ellipse x=${this.x} y=${this.y} width=${this.width} height=${this.height}]`;
+        return `[@pixi/math:Ellipse x=${this.x} y=${this.y} width=${this.halfWidth} height=${this.halfHeight}]`;
```

Comment:
If this has changed to half width/height then this log should also say halfWidth=...

Reply:
good catch! updated